bucket/picgo.json

```
@@ -0,0 +1,37 @@
+{
+    "version": "2.2.2",
+    "homepage": "https://picgo.github.io/PicGo-Doc/en/",
+    "description": "A simple & beautiful tool for pictures uploading built by vue-cli-electron-builder",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Molunerfinn/PicGo/releases/download/v2.2.2/PicGo-Setup-2.2.2.exe#/dl.7z",
+            "hash": "sha512:18144b18773c4ddb5aaebd3b51bc1e8c437deed80e0bdf1029d033d7374d56cc001052a4225a8901d3f85dd9408abf394cb7745de0164623240913cc7b9623d9",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse"
+            ]
+        }
+    },
+    "shortcuts": [
```

https://github.com/Molunerfinn/PicGo/blob/dev/src/main/utils/handleArgv.ts

```suggestion
    "bin": "PicGo.exe",
    "shortcuts": [
```